openprescribing/frontend/views/views.py

```diff
@@ -21,13 +21,10 @@
 from django.shortcuts import get_object_or_404, redirect, render
 from django.template.loader import render_to_string
 from django.urls import get_resolver, reverse
-from django.utils.http import is_safe_url
 from django.utils.safestring import mark_safe
 from dmd.models import VMP
-from frontend.feedback import send_feedback_mail
 from frontend.forms import (
     BookmarkListForm,
-    FeedbackForm,
```

Comment:
question: Can `FeedbackForm` be removed too?

https://github.com/ebmdatalab/openprescribing/blob/0344e66a785984ad4ca211a6b9600debe40a0aa9/openprescribing/frontend/forms.py#L102-L124

It only looks to be used here, and those uses are removed in this PR.

Reply:
Good catch, thanks. I thought I had removed this but must have missed it.